FX/bryanchi_FX Devices.lua

```diff
@@ -0,0 +1,24 @@
+-- @description FX Devices
+-- @author Bryan Chi
+-- @version 1.0beta
+-- @changelog
+--   --------------------------==  declare Initial Variables & Functions  ------------------------
+--       VersionNumber = 'Dec 23 2022 - bug fix 3 '
+--       FX_Add_Del
+-- @provides
+--   . > BryanChi_FX_Devices.lua
```

Comment:
Use the "Storage file name" field instead of "Install file name" to replace bryanchi to BryanChi.